pkgs/by-name/db/dbeaver-bin/package.nix

```diff
@@ -107,9 +110,13 @@ stdenvNoCC.mkDerivation (finalAttrs: {
         runHook postInstall
       '';
 
-  autoPatchelfIgnoreMissingDeps = [
-    "libc.so.8"
-  ];
+  preFixup = ''
+    # most directories are for different architectures, only keep what we need
+    shopt -s extglob
+    pushd $out/opt/dbeaver/plugins/com.sun.jna_5.15.0.v20240915-2000/com/sun/jna/
```

Comment:
For the Darwin system, the correct path is: `$out/Applications/dbeaver.app/Contents/Eclipse/plugins/com.sun.jna_5.15.0.v20240915-2000/com/sun/jna/`

Reply:
Thanks a lot for testing, I see in the `installPhase` where these 2 different locations are coming from. To write less platform dependent code, I moved it to `preInstall`, can you check it again?

Reply:
Darwin was still unhappy because there’s an `Eclipse` prefix in the path